e2e/web-e2e_test.go

```diff
@@ -0,0 +1,64 @@
+//go:build webassets_embed
+
+package integration
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/integration/helpers"
+	"github.com/gravitational/teleport/lib/auth/authclient"
+	"github.com/gravitational/teleport/lib/services"
+	"github.com/stretchr/testify/require"
+)
+
+// TestSignup sets up a test instance of Teleport and runs a playwright test against it to test the signup flow.
+func TestSignup(t *testing.T) {
+	setupAndRunBasicTest(t, "signup.spec.ts")
+}
+
+// TestCreateNewRole sets up a test instance of Teleport and a user to test role management in the UI.
+func TestRoleManagement(t *testing.T) {
+	setupAndRunBasicTest(t, "roles.spec.ts")
+}
+
+// TestAuthConnectorManagement sets up a test instance of Teleport and a user to test auth connector management in the UI.
+func TestAuthConnectorManagement(t *testing.T) {
+	setupAndRunBasicTest(t, "authconnectors.spec.ts")
+}
+
+// setupAndRunBasicTest sets up a test instance of Teleport and a user with the access and editor roles and runs a playwright test.
+// This is a helper function in cases where there is no additional backend setup required beyond creating an invite link.
+func setupAndRunBasicTest(t *testing.T, playwrightTest string) {
```

Comment:
I think it's a basic setup, not a basic test :p
```suggestion
func makeBasicSetupAndRunTest (t *testing.T, playwrightTest string) {
```